app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+
+from app.errors import (OutdatedVaccineError,
+                        NotVaccinatedError, NotWearingMaskError)
```

Comment:
imports should b estructured properly
```suggestion
from app.errors import (
    OutdatedVaccineError,
    NotVaccinatedError, 
    NotWearingMaskError,
)
```